app/components/index.ts

```diff
@@ -0,0 +1,5 @@
+import Button from "./Button";
+import TextInput from "./TextInput";
+import MuiIcons from "./MuiIcons";
+
+export { Button, TextInput, MuiIcons };
```

Comment:
to simplify this can we

```
export * from "./Button";
export * from "./TextInput";
export * from "./MuiIcons";
``` 

why import all? becuase in some case we need to call its props interface

Reply:
done